Formula/python@2.rb

```diff
@@ -92,12 +92,13 @@ def install
     if MacOS.sdk_path_if_needed
       # Help Python's build system (setuptools/pip) to build things on SDK-based systems
       # The setup.py looks at "-isysroot" to get the sysroot (and not at --sysroot)
-      cflags  << "-isysroot #{MacOS.sdk_path}" << "-I#{MacOS.sdk_path}/usr/include"
+      cflags  << "-isysroot #{MacOS.sdk_path}" << "-I#{MacOS.sdk_path}/usr/include" << "-I/usr/include"
```

Comment:
I've added `<< "-I/usr/include"` instead of replacing `"-I#{MacOS.sdk_path}/usr/include"` just in case. I'm pretty sure that `"-I#{MacOS.sdk_path}/usr/include"` is wrong, based on  setup.py of the Python 2.7.15, but it might be helpful elsewhere.